psst-gui/src/ui/playback.rs

```diff
@@ -186,14 +186,22 @@ fn player_widget() -> impl Widget<Playback> {
     Flex::row()
         .with_child(
             small_button_widget(&icons::SKIP_BACK)
-                .on_left_click(|ctx, _, _, _| ctx.submit_command(cmd::PLAY_PREVIOUS)),
-        )
+                .on_left_click(|ctx, _, now_playing: &mut Playback, _| {
+                    ctx.submit_command(cmd::PLAY_PREVIOUS);
+                    ctx.submit_command(lyrics::SHOW_LYRICS.with(now_playing.now_playing.as_ref().unwrap().clone()))
```

Comment:
@jacksongoode could you help me with this I cant figure out how to get it to load the lyrics when its skipped to the next track, With this it only does something after you click it twice

Reply:
Is this play_previous correct? I haven't dug into this but I would think we would need to listen for that command while on the lyrics page and the song changes we should fetch the new songs lyrics.

Reply:
I fixed this in the most recent commit! @SO9010